lib/resque/scheduler.rb

```diff
@@ -136,7 +136,7 @@ def load_schedule_job(name, config)
           interval_defined = false
           interval_types = %w(cron every)
           interval_types.each do |interval_type|
-            next unless !config[interval_type].nil? && !config[interval_type].empty?
+            next unless !config[interval_type].nil? && !config[interval_type].to_s.empty?
```

Comment:
What is this fixing?

Reply:
@iloveitaly I believe this is since the interval type can now be `Numeric` and `empty?` is a `String` only.